### PR TITLE
Increase timeout to wait after module upload in BO

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Increase timeout to wait after module upload in BO
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Sponsor company   | ~
| How to test?      | ~

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
